### PR TITLE
[jsk_fetch_startup] add rosbag_play.launch and rosbag_record.launch

### PR DESCRIPTION
**Pull request by knorth55**
originally from @sktometometo 
modified by @knorth55 

I add `rosbag_play.launch` and `rosbag_record.launch` for fetch.
in that point, we add `latched_topic_publisher.py` node for `/tf_static` and `/map` topics.

### CODE REVIEW

**Comment by knorth55** [comment]
you can try with following command and get the rviz output

```bash
roslaunch jsk_fetch_startup rosbag_play.launch rosbag:=<rosbag filepath> gui:=true
```

fetch_rosbag_kitchen

**Comment by 708yamaguchi** [comment]
I could use this pull request, too.
https://github.com/jsk-ros-pkg/jsk_recognition/issues/2509#issuecomment-641712633

**Comment by knorth55** [comment]
@k-okada can you please merge this PR?

File: jsk_fetch_robot/jsk_fetch_startup/launch/rosbag_play.launch [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  <node name="depth_image_relay" pkg="topic_tools" type="relay"
+        args="$(arg DEPTH_IMAGE) /head_camera/depth_registered/image_rect" />
+  <node name="depth_info_relay" pkg="topic_tools" type="relay"
+        args="$(arg DEPTH_CAMERA_INFO) /head_camera/depth_registered/camera_info" />
+  <node name="points_relay" pkg="topic_tools" type="relay"
+        args="/head_camera/depth_registered/throttled/points /head_camera/depth_registered/points" />
+  <node name="quat_points_relay" pkg="topic_tools" type="relay"
+        args="/head_camera/depth_registered/quater/throttled/points /head_camera/depth_registered/quater/points" />
+
+  <!-- latched topic republisher -->
```
Comment (708yamaguchi) [inline jsk_fetch_robot/jsk_fetch_startup/launch/rosbag_play.launch:77]: @knorth55 
Why do you republish latched topics?
Without this script, I can rostopic echo latched topic.

For example,
```bash
# terminal 1
roscore
# terminal 2
rosbag play 20200619100612_go_to_kitchen_rosbag.bag
# terminal 3
rostopic echo /map --noarr
header:                                            
  seq: 0                                           
  stamp:                                           
    secs: 1592506865                               
    nsecs: 636232232                               
  frame_id: "map"                                  
info:                                              
  map_load_time:                                   
    secs: 1592506865                               
    nsecs: 636231939                               
  resolution: 0.0500000007451                      
  width: 1200                                      
  height: 1200                                     
  origin:                                          
    position:                                      
      x: -13.0                                     
      y: -48.5                                     
      z: 0.0                                       
    orientation:                                   
      x: 0.0                                       
      y: 0.0                                       
      z: 0.0                                       
      w: 1.0                                       
data: "<array type: int8, length: 1440000>"        
---     


```

Comment (knorth55) [reply jsk_fetch_robot/jsk_fetch_startup/launch/rosbag_play.launch:77]: @sktometometo can you give us the situation which you need this node?

**Pull request merged**